haplotypecaller-gvcf-gatk4.wdl

```diff
@@ -88,77 +87,95 @@ workflow HaplotypeCallerGvcf_GATK4 {
 task HaplotypeCaller {
   File input_bam
   File input_bam_index
-  String gvcf_name
+  File interval_list
+  String vcf_basename
   File ref_dict
   File ref_fasta
   File ref_fasta_index
-  File interval_list
-  Int? interval_padding
   Float? contamination
-  Int? max_alt_alleles
```

Comment:
Was there a reason you dropped this?

Reply:
I am using the haplotypcaller task from eric's five-dollar-genome-pipeline as a template. The script made max_alt_alleles optional for gatk35 but hard set it to `3` for gatk4. 
https://github.com/broadinstitute/dsde-pipelines/blob/e3670631da087546baa8a5f7af13511b32df4700/tasks/germline_variant_discovery.wdl#L71